app/views/layouts/application.html.slim

```diff
@@ -44,3 +44,7 @@ html
         'Created with
         i.fa.fa-heart
         'by the SUSE team
+      p
+        - if current_user.admin?
+          - if Version.value.present?
```

Comment:
Since this function might return nil, you can merge both if's as:

    - if current_user.admin? && Version.value